db_service/migrations.go

```diff
@@ -212,11 +212,11 @@ func RunMigrations(db *gorm.DB) error {
 
 	// if we've run any migrations before, we should have a migrations table, so find the last one we ran
 	if db.HasTable("migrations") {
-		lastMigration := models.Migration{}
-		if err := db.Order("created_at desc").First(&lastMigration).Error; err != nil {
+		var storedMigrations []models.Migration
+		if err := db.Order("id desc").Find(&storedMigrations).Error; err != nil {
```

Comment:
does this ID correspond to the key of [this map](https://github.com/GoogleCloudPlatform/gcp-service-broker/blob/master/db_service/migrations.go#L36)? Or is it incremented by mysql? If it's the former I recommend either controlling it or adding a new field that is controlled by you for sorting.